mava/environment_loop.py

```diff
@@ -245,14 +247,102 @@ def should_run_loop(eval_interval_condition: Tuple) -> bool:
             )
             evaluation_duration = eval_duration_condition[1]
 
-        def step_executor() -> None:
+        def run_evaluation(results: Any) -> None:  # pragma: no cover
+            """Calculate the absolute metric"""
+
+            normalisation = (
+                self._executor.has(ObservationNormalisation)
+                and self._executor.store.global_config.normalise_observations
+            ) or (
+                self._executor.has(ValueNormalisation)
+                and self._executor.store.global_config.normalise_target_values
+            )
+
+            if normalisation:
+                warnings.warn(
```

Comment:
Why does do we need a different computation when normalisation is applied?

Reply:
Yes unfortunately, from what we found that storing the params of the network related to norm is not the same as non normalized ones therefore we can't calculate the absolute metric or store  the best checkpointed params using the same util functions (that's why we create an issue for that https://github.com/instadeepai/Mava/issues/859 )

Reply:
I may create a PR for it before the end of the week!